src/data/valid/Database-nucleic-extraction.yaml

```diff
@@ -13,7 +13,7 @@ extraction_set:
     #      the range of has_input for this class, as a slot_usage
     # this still doesn't solve referential integrity
     has_input:
-      - generic:xxx
+      - nmdc:procsm-99-123456
 
     #cardinality of outputs should be 1-many, may want to fractionate samples or keep supernatant
     has_output:
```

Comment:
Same as PlannedProcess subclasses, range for has_output needs to be updated to PlannedProcess and example data needs to be updated.